### PR TITLE
Converted everything to hooks and to styled-components

### PR DESCRIPTION
**Pull request by tcoldx**

### CODE REVIEW

File: src/Pages/Coin/Coin.js [hunk 1]
```diff
@@ -120,135 +120,131 @@ const checkIfNull = (value) => {
...
-        {coin && (
-          <div>
-            <Container>
-              <CoinName>
-                <ImageTheme src={coin.image.small} alt="sora" />
-                <span>{coin.name}</span>
-                <span>({coin.symbol.toUpperCase()})</span>
-              </CoinName>
-              <CoinName>
+  } */
```
Comment (yoloOnTheBattlefield) [inline src/Pages/Coin/Coin.js:139]: remove old code

File: src/Pages/Coin/Coin.js [hunk 2]
```diff
@@ -120,135 +120,131 @@ const checkIfNull = (value) => {
...
+          <Website>
+            <Link href={coin.links.homepage[0]}>{coin.links.homepage[0]}</Link>
+          </Website>
+          <Body>Description</Body>
+          <DescriptionContainer>
+            <p
+              dangerouslySetInnerHTML={{
+                __html: `<p>${coin.description.en}</p>`,
+              }}
+            ></p>
```
Comment (yoloOnTheBattlefield) [inline src/Pages/Coin/Coin.js:242]: why not:
```
<div
              dangerouslySetInnerHTML={{
                __html: coin.description.en,
              }}
            />```

File: src/Pages/Home/Home.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import CoinChart from "../../components/CoinChart/index";
+import CoinList from "../../components/CoinList/index";
```
Comment (yoloOnTheBattlefield) [inline src/Pages/Home/Home.js:4]: setup your folder structure bro

File: src/components/CoinChart/CoinChart.js [hunk 4]
```diff
@@ -1,141 +1,134 @@
...
-  };
+const Charts = styled.div`
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+  width: 1119.2px;
+  margin: 20px;
+`;
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinChart/CoinChart.js:21]: styles should be in a separate file

File: src/components/CoinChart/CoinChart.js [hunk 5]
```diff
@@ -1,141 +1,134 @@
...
-  }
-}
+  return (
+    <Charts>
+      <ChartContainer>
+        {/*<CoinLegendOne currency={props.currency} />*/}
+        <Line data={data} options={options} />
+      </ChartContainer>
+      <ChartContainer>
+        {/*<CoinLegendTwo currency={props.currency} />*/}
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinChart/CoinChart.js:127]: do you need this?

File: src/components/CoinTable/CoinTable.js [hunk 6]
```diff
@@ -1,86 +1,22 @@
...
+  CellThree,
+  CellFour,
+  Row,
+  Red,
+  Green,
+  PercentageBar,
+  Market,
+  Supply,
+  Price,
+} from "../CoinTable/CoinTable.styles";
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinTable/CoinTable.js:19]: setup your folder structure

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

**Review by tcoldx — COMMENTED** [review]
changed some of the files around and fixed the folder structure

File: src/App.js [hunk 7]
```diff
@@ -1,7 +1,7 @@
 import React, { useState } from "react";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
-import { NavBar, Stats } from "./components";
-import { Home, Coin, Portfolio } from "./Pages";
+import { NavBar, Stats } from "components";
+import { Home, Coin, Portfolio } from "Pages";
```
Comment (yoloOnTheBattlefield) [inline src/App.js:4]: `pages` should be lower case, you will have git problems after renaming
https://stackoverflow.com/questions/10523849/changing-capitalization-of-filenames-in-git

File: src/Pages/Coin/Coin.styles.js [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+  display: flex;
+  justify-content: space-between;
+  gap: 20px;
+`;
+export const Link = styled.a`
+  color: white;
+  font-size: 10px;
+  text-decoration: none;
+  font-weight: bold;
+`;
```
Comment (yoloOnTheBattlefield) [inline src/Pages/Coin/Coin.styles.js:97]: some variables have new line, some don't. choose one

File: src/Pages/Home/Home.js [hunk 9]
```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect } from "react";
 import axios from "axios";
-import CoinChart from "../../components/CoinChart/index";
-import CoinList from "../../components/CoinList/index";
+import CoinChart from "components/CoinChart/CoinChart";
+import CoinList from "components/CoinList/CoinList";
```
Comment (yoloOnTheBattlefield) [inline src/Pages/Home/Home.js:4]: import { CoinChart, CoinList } from 'components'

**Pull request merged**